doc/wg/core/notes/core-notes-2024-01-19.md

```diff
@@ -0,0 +1,129 @@
+# Tock Meeting Notes 1/19/24
+
+## Attendees
+- Alex Radovici
+- Amit Levy
+- Andrew Imwalle
+- Brad Campbell
+- Hudson Ayers
+- Johnathan Van Why
+- Leon Schuermann
+- Pat Pannuto
+- Tyler Potyondy
```

Comment:
Alyssa attended as well.

Reply:
Good catch. Sorry for leaving you off the list Alyssa!